Utils/update_contribution_pack_in_base_branch.py

```diff
@@ -18,22 +19,24 @@ def main():
     pr_number = args.pr_number
     repo = args.contrib_repo
     branch = args.branch
-    pack_dir_name = get_pack_dir(branch, pr_number, repo)
-    if not pack_dir_name:
+    packs_dir_names = get_pack_dir(branch, pr_number, repo)
+    if not packs_dir_names:
         print_error('Did not find a pack in the PR')
         sys.exit(1)
-
-    pack_dir = f'Packs/{pack_dir_name}'
+    print(f'Copy changes from the contributor branch {repo}/{branch} '
+          f'in the following packs: ' + '\n'.join(packs_dir_names))
 
     try:
-        if os.path.isdir(pack_dir):
-            # Remove existing pack
-            shutil.rmtree(pack_dir)
+        for pack_dir in packs_dir_names:
+            if os.path.isdir(f'Packs/{pack_dir}'):
+                # Remove existing pack
+                shutil.rmtree(f'Packs/{pack_dir}')
+        string_dir_names = f'Packs/{" Packs/".join(packs_dir_names)}'
```

Comment:
Le'ts just add an explaining comment:

```suggestion
        # if packs_dir_names = ['pack_a', 'pack_b', 'pack_c'], string_dir_names will be 'Packs/pack_a Packs/pack_b Packs/pack_c'
        string_dir_names = f'Packs/{" Packs/".join(packs_dir_names)}'
```